src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -532,4 +538,14 @@ public function setResourcesPath($resourcesPath)
     {
         $this->addResourcePath($resourcesPath);
     }
+
+    /**
+     * Caster that removes all internals.
+     *
+     * @see AbstactCloner
+     * @return array
+     */
+    public function castObject($obj, $a, $stub, $isNested) {
```

Comment:
Could we use a closure instead so we dont need this additional public api method?